### PR TITLE
Document 'download' query parameter

### PR DESCRIPTION
**Pull request by alyssaruth**
Documents the `download` query parameter for downloading files from a given release. I've updated two places:

 - Project release file [here](https://docs.sentry.io/api/releases/retrieve-a-project-releases-file/). 
 - Organization release file [here](https://docs.sentry.io/api/releases/retrieve-an-organization-releases-file/)

I've assumed it works for both, but have only personally verified the first one in testing so worth checking I'm not lying about the organization one :grin: 

This was mentioned in [this issue comment](https://github.com/getsentry/sentry/issues/32107#issuecomment-1060521498) but never actually done. Also mentioned in https://github.com/getsentry/sentry/issues/42759.

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: api-docs/paths/releases/project-release-file.json [hunk 1]
```diff
@@ -116,6 +116,14 @@
...
           "type": "string"
         }
+      },
+      {
+        "name": "download",
+        "in": "query",
+        "description": "If this is set to true, then the response payload will be the raw file contents. Otherwise, the response will be the file metadata as JSON.",
+        "schema": {
+          "type": "boolean"
+        }
```
Comment (markstory) [inline api-docs/paths/releases/project-release-file.json:126]: Should this parameter be in the GET method? Currently you have it in the PUT method.

File: api-docs/paths/releases/release-file.json [hunk 2]
```diff
@@ -30,6 +30,14 @@
         "schema": {
           "type": "string"
         }
+      },
+      {
+        "name": "download",
```
Comment (markstory) [inline api-docs/paths/releases/release-file.json:35]: Organization Release File Details does support the `download` parameter.

File: api-docs/paths/releases/project-release-file.json (hunk 1 above)
Comment (alyssa-glean) [reply api-docs/paths/releases/project-release-file.json]: :facepalm: yes, I've moved it!

**Review by markstory — APPROVED** [review]

**Comment by markstory** [comment]
Thank you 🎉

**Pull request merged**